openapi/v2/core-metadata.yaml

```diff
@@ -2227,6 +2277,421 @@ paths:
               examples:
                 500Example:
                   $ref: '#/components/examples/500Example'
+  '/deviceprofile/basicinfo':
+    parameters:
+      - $ref: '#/components/parameters/correlatedRequestHeader'
+    patch:
+      summary: "Allows basic information updates to an existing device profile"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/DeviceProfileBasicInfoRequest'
+            example:
```

Comment:
provide the example only modifying partial fields

Reply:
updated